docs/src/components/HomepageFeatures/python.md

```diff
@@ -1,7 +1,7 @@
 ## Python
 
 \`perspective-python\`, built on the same C++ data engine used by the
-[WebAssembly version](https://perspective.finos.org/docs/md/js.html), implements
+[WebAssembly version](https://perspective.finos.org/docs/obj/perspective-python/), implements
```

Comment:
This seems like the wrong link. It used to link to a `/js` page but now `/python`

Reply:
The page demands link to be directed to perspective-python api doc.
The current https://perspective.finos.org/docs/md/js.html gives error.
If it shoud direct to another relevent link please give it.

Reply:
The [actual Python documentation](https://perspective.finos.org/docs/python/) has similar copy at the top, and links to [The JS docs](https://perspective.finos.org/docs/js/) (https://perspective.finos.org/docs/js/). So I think we should link to the JS docs there. 

However, it probably is a good idea to link to the Python docs here, probably make the `perspective-python` at the start of the paragraph a link to https://perspective.finos.org/docs/python/, and not the API docs themselves?

Regardless of that though, this should be a link to the JS docs, as the Python docs do.